### PR TITLE
feat(music): Bump compatibility to v5.41.50

### PR DESCRIPTION
**Pull request by validcube**
✨ Feature - YouTube Music - Bump compatibility to v5.41.50

Tested compiling on:

- [x] `ReVanced CLI` - v2.20.0
> ✅ Success - All patches have been correctly installed.
- [x] `ReVanced Manager` - v0.0.53
> ✅ Success - All patches have been correctly installed.

I have tested these features on my device and have confirmed that it is working fine.

- [x] `background-play`
- [x] `codecs-unlock`
- [x] `compact-header`
- [x] `exclusive-audio-playback`
- [x] `hide-get-premium`
- [x] `minimized-playback-music`
- [x] `music-microg-support`
- [x] `music-video-ads`
- [x] `tasteBuilder-remover`
- [x] `upgrade-button-remover`

### CODE REVIEW

**Comment by FineFindus** [comment]
Did you check if the `hide-getpremium` patch works? I didn't update the compatibility last time, since I did not always work, and sometimes premium ads still showed up.

**Comment by EvadeMaster** [comment]
@FineFindus Yes, I have checked that `hide-get-premium` does work on my device.

_Sorry about that slow response, I kid you not, I didn't hit "Comment"._

**Comment by oSumAtrIX** [comment]
@EvadeMaster, I can not reproduce an issue with `hide-get-premium`. How do you reproduce? @FineFindus

**Comment by FineFindus** [comment]
I simply opened the app and it had a full-screen premium ad. But it's not reliable, since I didn't get one every time.

**Comment by oSumAtrIX** [comment]
In that case it should be removed from the bump

File: src/main/kotlin/app/revanced/patches/music/layout/premium/annotations/HideGetPremiumCompatibility.kt [hunk 1]
```diff
@@ -23,7 +23,8 @@ import app.revanced.patcher.annotation.Package
             "5.34.51",
             "5.36.51",
             "5.38.53",
-            "5.39.52"
+            "5.39.52",
+            "5.41.50"
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/music/layout/premium/annotations/HideGetPremiumCompatibility.kt:27]:
```suggestion
            "5.39.52"
```

**Comment by theRookieCoder** [comment]
The readme hasn't been updated yet. Every patch has to be checked for compatibility right?

**Comment by oSumAtrIX** [comment]
The readme suggests the most recommended version, it is updated.

**Comment by theRookieCoder** [comment]
The [release](https://github.com/revanced/revanced-patches/releases/tag/v2.159.0) suggests that the version has been bumped, but neither the actual patches nor the readme have been updated.

**Comment by oSumAtrIX** [comment]
The release suggests it correctly. Patches have been bumped, but not all. The recommended version is calculated based on which version supports most patches.

**Comment by oSumAtrIX** [comment]
You can simply select another version to patch. ReVanced doesn't really restrict you even selecting incompatible versions.

**Comment by theRookieCoder** [comment]
Hmm alright but shouldn't the readme specify the latest supported version since they're specified individually for each patch anynways.

**Comment by oSumAtrIX** [comment]
That's a good point. Please open an issue regarding this.

**Comment by theRookieCoder** [comment]
Could #1584 be relevant?

**Comment by oSumAtrIX** [comment]
Yes, but it should be opened in the patches repository.

**Pull request merged**